spec/helpers/helpers.rb

```diff
@@ -0,0 +1,14 @@
+module Helpers
```

Comment:
I would move this file to `spec/support/features/authentication_helpers.rb`

The reason we have the `spec/support` directory is to put helper files in it.
And since this should be feature helper methods, regarding authentication, it makes sense to call it after what it does.

Reply:
Done in sha: 5e2fd92